src/main/java/com/reshiftsecurity/plugins/intellij/gui/editor/BugAnnotator.java

```diff
@@ -202,18 +202,20 @@ private static void addAnnotation(final ExtendedProblemDescriptor problemDescrip
 	}
 
 	private static String getAnnotationText(final List<ExtendedProblemDescriptor> problemDescriptors) {
+		EducationCachingService _eduCacheService = ServiceManager.getService(EducationCachingService.class);
 		final StringBuilder buffer = new StringBuilder();
-		for (//noinspection LocalCanBeFinal
-				int i = 0, problemDescriptorsSize = problemDescriptors.size(); i < problemDescriptorsSize; i++) {
-			final ExtendedProblemDescriptor problemDescriptor = problemDescriptors.get(i);
-			buffer.append(ResourcesLoader.getString("findbugs.name")).append(": ").append(StringUtilFb.html2text(BugInstanceUtil.getBugPatternShortDescription(problemDescriptor.getBug().getInstance()))).append('\n');
-			buffer.append(StringUtilFb.html2text(BugInstanceUtil.getDetailText(problemDescriptor.getBug().getInstance())));
-			if (i < problemDescriptors.size() - 1) {
-				//noinspection HardcodedLineSeparator
-				buffer.append("\n\n");
-			}
-		}
-
+		buffer.append(_eduCacheService.getBriefOverview(problemDescriptors, true));
```

Comment:
is this protected against xss?

Reply:
Will add sanitization in `getBriefOverview`